src/ekat/util/ekat_units.hpp

```diff
@@ -97,23 +110,23 @@ class Units {
 
 private:
 
-  friend bool operator==(const Units&,const Units&);
+  friend constexpr bool operator==(const Units&,const Units&);
 
-  friend Units operator*(const Units&,const Units&);
-  friend Units operator*(const ScalingFactor&,const Units&);
-  friend Units operator/(const Units&,const Units&);
-  friend Units operator/(const Units&,const ScalingFactor&);
-  friend Units operator/(const ScalingFactor&,const Units&);
-  friend Units pow(const Units&,const RationalConstant&);
-  friend Units sqrt(const Units&);
-  friend Units root(const Units&,const int);
+  friend constexpr Units operator*(const Units&,const Units&);
+  friend constexpr Units operator*(const ScalingFactor&,const Units&);
+  friend constexpr Units operator/(const Units&,const Units&);
+  friend constexpr Units operator/(const Units&,const ScalingFactor&);
+  friend constexpr Units operator/(const ScalingFactor&,const Units&);
+  friend constexpr Units pow(const Units&,const RationalConstant&);
+  friend constexpr Units sqrt(const Units&);
+  friend constexpr Units root(const Units&,const int);
 
   friend std::string to_string(const Units&);
 
   const ScalingFactor     m_scaling;
   const RationalConstant  m_units[7];
```

Comment:
Seems like Weaver is having trouble copying this ^^^ member in one of your `constexpr` constructors. Maybe it needs to be copied within the constructor body?